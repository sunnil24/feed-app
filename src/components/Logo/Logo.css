.logo {
  border: 1px solid #fff;
  padding: 0;
  height: 18px;
  overflow: hidden;
  line-height: 0;
}
