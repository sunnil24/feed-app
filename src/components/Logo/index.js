import React from "react";
import { Link } from "react-router-dom";

// styles
import "./Logo.css";

//logo
import logo from "../../assests/images/y18.gif";

const Logo = () => {
  return (
    <h1 className="logo">
      <Link to="/" title="Hacker News | Feed">
        <img src={logo} alt="Hacker News" />
      </Link>
    </h1>
  );
};

export default Logo;
