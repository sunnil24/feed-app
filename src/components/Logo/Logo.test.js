import React from "react";
import { shallow } from "enzyme";

import Logo from "./index";

let wrapped = shallow(<Logo />);

describe("<Logo />", () => {
  it("should render the Title Component correctly", () => {
    expect(wrapped).toMatchSnapshot();
  });
});
