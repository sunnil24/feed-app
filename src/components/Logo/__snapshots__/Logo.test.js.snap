// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Logo /> should render the Title Component correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Logo />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <ForwardRef
        title="Hacker News | Feed"
        to="/"
      >
        <img
          alt="Hacker News"
          src="y18.gif"
        />
      </ForwardRef>,
      "className": "logo",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": <img
          alt="Hacker News"
          src="y18.gif"
        />,
        "title": "Hacker News | Feed",
        "to": "/",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "alt": "Hacker News",
          "src": "y18.gif",
        },
        "ref": null,
        "rendered": null,
        "type": "img",
      },
      "type": Object {
        "$$typeof": Symbol(react.forward_ref),
        "displayName": "Link",
        "propTypes": Object {
          "innerRef": [Function],
          "onClick": [Function],
          "replace": [Function],
          "target": [Function],
          "to": [Function],
        },
        "render": [Function],
      },
    },
    "type": "h1",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <ForwardRef
          title="Hacker News | Feed"
          to="/"
        >
          <img
            alt="Hacker News"
            src="y18.gif"
          />
        </ForwardRef>,
        "className": "logo",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "function",
        "props": Object {
          "children": <img
            alt="Hacker News"
            src="y18.gif"
          />,
          "title": "Hacker News | Feed",
          "to": "/",
        },
        "ref": null,
        "rendered": Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "alt": "Hacker News",
            "src": "y18.gif",
          },
          "ref": null,
          "rendered": null,
          "type": "img",
        },
        "type": Object {
          "$$typeof": Symbol(react.forward_ref),
          "displayName": "Link",
          "propTypes": Object {
            "innerRef": [Function],
            "onClick": [Function],
            "replace": [Function],
            "target": [Function],
            "to": [Function],
          },
          "render": [Function],
        },
      },
      "type": "h1",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromError": true,
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
    Symbol(enzyme.__providerValues__): undefined,
  },
  Symbol(enzyme.__providerValues__): Map {},
}
`;
